src/main/java/com/dev/cinema/controllers/InjectData.java

```diff
@@ -0,0 +1,25 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.Role;
+import com.dev.cinema.service.RoleService;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class InjectData {
```

Comment:
let's rename this class to DataInitializer and use the annotation Component instead of Controller